### PR TITLE
[dvs] Upload logs even if failure occurs during startup

### PR DESCRIPTION
**Pull request by daall**
Signed-off-by: Danny Allen <daall@microsoft.com>

**What I did**
I updated `dvs` to use the module log path even if the failure occurs during virtual switch setup.

**Why I did it**
Previously the `get_logs()` call in `check_ready_status` passed None as the `modpath`, which means any failure during startup would reset the logs. This makes it hard to debug startup issues for particular test cases, and it deletes a lot of helpful data from other non-startup related test failures.

**How I verified it**
Inserted `assert 2 == 3` into the DockerVirtualSwitch constructor, verified that logs were output to `test_speed` (no dvs specified) and `my_virtual_switch` (persistent container specified) respectively.

**Details if related**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -970,12 +974,14 @@ def dvs(request) -> DockerVirtualSwitch:
...
+
+    dvs = DockerVirtualSwitch(name, imgname, keeptb, fakeplatform, log_path)
+
     yield dvs
-    if name == None:
-        dvs.get_logs(request.module.__name__)
-    else:
-        dvs.get_logs()
+
+    dvs.get_logs()
```
Comment (lguohan) [inline tests/conftest.py:984]: I did not realize we only capture the syslog, we should probably dump the db and capture that as well (not in this pr though)

Comment (daall) [reply tests/conftest.py:984]: We do also get sairedis.rec and supervisor logs as well, but I think dumping the DB should also help

Comment (lguohan) [reply tests/conftest.py:984]: does show tech work in the vs docker? if it works, then we can use that, that captures every thing.

Comment (daall) [reply tests/conftest.py:984]: It looks like it fails because `sudo` is not found. I'll dig a little deeper and see if we can make it work

**Comment by daall** [comment]
retest vs please

**Pull request merged**